mobile-app/app/screens/AppNavigator/screens/Portfolio/components/PortfolioCard.tsx

```diff
@@ -122,25 +101,31 @@ export function PortfolioCard ({
         )
       }
       <View testID='card_balance_row_container'>
-        {filteredTokens.map((item) => (
-          <View key={item.symbol} style={tailwind('p-4 pt-1.5 pb-1.5')}>
-            <PortfolioItemRow
-              onPress={() => navigation.navigate({
-                name: 'Balance',
-                params: { token: item, usdAmount: item.usdAmount },
-                merge: true
-              })}
-              token={item}
-              denominationCurrency={denominationCurrency}
-            />
-          </View>
-        ))}
+        {filteredTokens.length > 0
+        ? (
+          <>
+            {filteredTokens.map((item) => (
+              <View key={item.symbol} style={tailwind('p-4 pt-1.5 pb-1.5')}>
```

Comment:
```suggestion
              <View key={item.symbol} style={tailwind('p-4 py-1.5')}>
```

Reply:
this is replaced by other component after merge from main